src/accounts/views.py

```diff
@@ -71,9 +85,35 @@ def form_valid(self, form):
 
     @staticmethod
     def create_profile(user=None, **kwargs):
+
         # Creates a new UserProfile object after successful creation of User object
         userprofile = CompanyProfile.objects.create(user=user, name=kwargs['name'], domain=kwargs['domain'],
                                                     url=kwargs['url'], city=kwargs['city'], state=kwargs['state'],
                                                     country=kwargs['country'], pin_code=kwargs['pin_code'],
                                                     contact=kwargs['contact'],)
         userprofile.save()
+
+        # Sends email to the registered company
+        subject = "Registered with SPC"
+        from_email = settings.EMAIL_HOST_USER
+        to_email = [user.username, ]
+        with open(settings.BASE_DIR + "/templates/accounts/company_register_email.txt") as f:
+            register_message = f.read()
+        message = EmailMultiAlternatives(subject=subject, body=register_message, from_email=from_email, to=to_email)
+        html_template = render_to_string("accounts/company_register_email.html", {'username': user.username})
+        message.attach_alternative(html_template, "text/html")
+        message.send()
+
+        # Sends details of the registered company
+        subject1 = "Company Registered with SPC"
+        from_email1 = settings.EMAIL_HOST_USER
+        to_email1 = [settings.TO_EMAIL, ]
+        with open(settings.BASE_DIR + "/templates/accounts/company_details_email.txt") as f1:
+            details_message = f1.read()
+        message1 = EmailMultiAlternatives(subject=subject1, body=details_message, from_email=from_email1, to=to_email1)
+        html_template1 = render_to_string("accounts/company_details_email.html", {'name': kwargs['name'], 'email': user.username,
```

Comment:
if you move it to a seperate function, you won't need to use `message1` and `html_template1`